zerver/lib/cache.py

```diff
@@ -534,6 +534,10 @@ def realm_user_dicts_cache_key(realm_id: int) -> str:
     return f"realm_user_dicts:{realm_id}"
 
 
+def get_muting_users_cache_key(muted_user_profile: "UserProfile") -> str:
```

Comment:
I guess I didn't update this commit for the `muted_user_profile => muted_user` rename; can you take care of that?